### PR TITLE
[BACK-35] Scraping architecture

### PR DESCRIPTION
**Pull request by DogukanGun**
## Description of Issue
General web scraping architecture

## How I implemented
I created an interface which should be used for all kinds of scraping. Then the web scraping should be added into the _scrapers list where is stored into src/webscraping/bridge/WebScrapingService.ts file.

### CODE REVIEW

**Review by thomas-woehrle — COMMENTED** [review]
We don't need this. A lot of scraping functionality is already implemented and won't switch to this structure.

File: src/webscraping/scrapers/IScraper.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { WebData } from '../bridge/WebData'
+
+export interface IScraper {
```
Comment (pczern) [inline src/webscraping/scrapers/IScraper.ts:3]: our conventions are to avoid interfaces

**Comment by pczern** [comment]
@DogukanGun please check with @thomas-woehrle, share the result in this PR either merge after review or close
tests are failing, converting to Draft

**Comment by DogukanGun** [comment]
> @DogukanGun please check with @thomas-woehrle, share the result in this PR either merge after review or close
> tests are failing, converting to Draft

Okay this decision belongs to backend team therefore I closed this pr.

**Pull request closed without merging**